vignettes/articles/cookbook_advanced_use_cases.Rmd

```diff
@@ -342,27 +349,25 @@ ei_sector_targets <- sda_targets_portfolio %>%
 available_sectors <- unique(pull(filter(matched, sector %in% c(unique(scenario$sector), unique(co2$sector))), sector))
 
 ```
-In this example, we set targets for a bank, based on its loan book composition in `r min(scenario$year, na.rm = TRUE)`. The loan book contains exposures to the following PACTA sectors, for which trajectories are available in the scenarios: `r paste(available_sectors, collapse = ", ")`. The bank wants to set targets for the year 2030. The bank has decided to use the "target_sda" scenario as a benchmark for setting targets based on the [market share approach](cookbook_metrics.html#market-share-approach) and "target_demo" for setting targets based on the [sectoral decarbonization approach](cookbook_metrics.html#sectoral-decarbonization-approach). Targets are set based on global scenario benchmarks. The following two tables show how targets are set for sectors based on PACTA.
 
-```{r use_case_two_sector_targets_results, echo = FALSE, warning = FALSE}
-result_table(tech_mix_sector_targets, title = "Sector targets", subtitle = "high-carbon share")
+In this example, we set targets for a bank, based on its loan book composition in `r min(scenario$year, na.rm = TRUE)`. The loan book contains exposures to the following PACTA sectors, for which trajectories are available in the scenarios: `r paste(available_sectors, collapse = ", ")`. The bank wants to set targets for the year 2030. The bank has decided to use the "target_sda" scenario as a benchmark for setting targets based on the [market share approach](cookbook_metrics.html#market-share-approach) and "target_demo" for setting targets based on the [sectoral decarbonization approach](cookbook_metrics.html#sectoral-decarbonization-approach). Targets are set based on global scenario benchmarks. The sector targets for 2030 based on this loan book are as follows:
 
-result_table(ei_sector_targets, title = "Sector targets", subtitle = "emission intensity")
+```{r use_case_two_sector_targets_high_carbon, echo = FALSE, warning = FALSE}
+result_table(high_carbon_share_sector_targets, title = "Sector targets", subtitle = "high-carbon share")
 ```
 
-The sector targets for 2030 based on this loan book are as follows:
+For ssectors with technology pathways, the banks could set targets based on the high-carbon share of technologies in the sector. Alternatively it could break down the targets by individual technologies.
```

Comment:
```suggestion
For sectors with technology pathways, the banks could set targets based on the high-carbon share of technologies in the sector. Alternatively it could break down the targets by individual technologies.
```